yak/package.xml

```diff
@@ -11,7 +11,8 @@
   <depend>libpcl-all-dev</depend>
   <depend>eigen</depend>
   <depend>nvidia-cuda-dev</depend>
-  <depend>libopencv-dev</depend>
+  <depend>opencv3</depend> <!-- key for Xenial -->
```

Comment:
This is not a valid `rosdep` key: see [ros/rosdistro/rosdep/base.yaml](https://github.com/ros/rosdistro/blob/5a384b4bc7fdb0fa0a5ab211a42bcc63a5964e5d/rosdep/base.yaml) and search for `opencv3`.

For all ROS versions, `libopencv-dev` is the correct key, and should resolve to the OpenCV version distributed by the OS.